src/main/resources/application.properties

```diff
@@ -61,3 +61,4 @@ oauth.redirectUris=http://localhost
 oauth.clientId=3bc8282c-7730-11e5-8bcf-feff819cdc9f
 oauth.clientSecret=6035bd78-7730-11e5-8bcf-feff819cdc9f
 unitDatabase.url=http://spooky.github.io/unitdb/#/
+clan.clanWebpagesBaseUrl=http://clans.faforever.com/clan_
```

Comment:
The prefered way is to use a string format:
`http://clans.faforever.com/_%s` since it's more self-explanatory and can more easily be adjusted if the clan ID isn't at the end anymore.
The code which uses it needs to use `String.format()`